teams/views.py

```diff
@@ -83,28 +88,33 @@ class MyTeamsListView(ListView):
     model = Team
 
     def get(self, request):
-        team_list = TeamInvite.objects.filter(user=self.request.user, accepted=True)
+        team_list = Team.objects.filter(
+            Q(captains__exact=request.user) | Q(founder=request.user) | Q(players__exact=request.user))
+        # team_list = TeamInvite.objects.filter(user=self.request.user, accepted=True)
         return render(request, 'teams/team_list.html', {'team_list': team_list})
 
     def get_queryset(self, **kwargs):
-        # TO DO switch the filter to the players field not just the founder field.
-        if TeamInvite.objects.filter(user=self.request.user, accepted=True):
-            # TO DO switch the filter to the players field not just the founder field.
-            return TeamInvite.objects.filter(user=self.request.user, accepted=True)
+        return Team.objects.filter(
+            Q(captains__exact=self.request.user) | Q(founder=self.request.user) | Q(players__exact=self.request.user))
 
 
 def edit_team_view(request, pk):
     if request.method == 'POST':
         teamobj = get_object_or_404(Team, id=pk)
         form = EditTeamProfileForm(request.POST, request.FILES, instance=teamobj)
+        if request.user in teamobj.captains or request.user is teamobj.founder:
+            pass
+        else:
+            messages.error(request, 'ERROR: You must be a captain or founder update team info')
+            return redirect('teams:detail', pk=teamobj.pk)
         if form.is_valid():
-            #teamobj.about_us = form.data['about_us']
-            #teamobj.website = form.data['website']
-            #teamobj.twitter = form.data['twitter']
-            #teamobj.twitch = form.data['twitch']
-            #teamobj.country = form.data['country']
-            #teamobj.image = form.data['image']
-            #teamobj.save()
+            # teamobj.about_us = form.data['about_us']
+            # teamobj.website = form.data['website']
+            # teamobj.twitter = form.data['twitter']
+            # teamobj.twitch = form.data['twitch']
+            # teamobj.country = form.data['country']
+            # teamobj.image = form.data['image']
+            # teamobj.save()
```

Comment:
if these commented lines aren't needed they should be deleted

Reply:
been there for quite some time. but I'll remove it